### PR TITLE
Promoting member to organizer

### LINKED ISSUE #137: Organization: promoting member to organizer
```

@organization
Feature:
  As an organizer
  I can promote other members to organizers

  Background:
    Given there is a "Local organization" organization created by "Alex Smith"
    And "Jo Johnson" is member of "Local organization" organization

  Scenario: Organizer can promote members to organizers
    Given I am logged in as "Alex Smith"
    When I promote "Jo Johnson" to organizer of "Local organization"
    Then "Jo Johnson" is organizer of "Local organization"

```

### PR DESCRIPTION
**Pull request by msvrtan**
In order to allow promoting existing member to an organizer, we need to support this feature

Key difference between adding an organizer and promoting is that for promotion, we expect user to already be a member!

Closes #137

### CODE REVIEW

File: src/Organization/Behat/OrganizationApplicationContext.php [hunk 1]
```diff
@@ -272,4 +273,13 @@ public function iWillGetAnErrorSayingThatUserIsAlreadyAnOrganizer()
...
         Assert::isInstanceOf($this->exception, DomainException::class);
     }
+
+    /**
+     * @Then I will get an error saying that user needs to be a member in order to be promoted
+     */
+    public function iWillGetAnErrorSayingThatUserNeedsToBeMemberInOrderToBePromoted()
+    {
+        Assert::notNull($this->exception);
+        Assert::isInstanceOf($this->exception, DomainException::class);
```
Comment (HcsOmot) [inline src/Organization/Behat/OrganizationApplicationContext.php]: Isn't there a way to actually check if the exception has the right message?
Something like:
```
Assert::contains("Actual Exception Message Here", $this->exception->getMessage());
```

Or - we could create a more precise and appropriate Exception, something like

```
- UserNotOrganizationMember::class
- CannotPromoteNonMember::user(UserEntity $user)
```

and then perform a more thorough check:
```
Assert::isInstanceOf($this->exception, UserNotOrganizationMember::class | CannotPromoteNonMember::class);
```

File: features/managing-organization-organizers.feature [hunk 2]
```diff
@@ -0,0 +1,13 @@
+@organization
+Feature:
+  As an organizer
+  I can promote other members to organizers
+
+  Background:
+    Given there is a "Local organization" organization created by "Alex Smith"
```
Comment (HcsOmot) [inline features/managing-organization-organizers.feature]: Ninjas be like
http://cf.chucklesnetwork.com/items/3/2/8/4/3/original/hiding-in-plain-sight-nailed-it.jpg

File: src/Organization/Behat/OrganizationApplicationContext.php (hunk 1 above)
Comment (msvrtan) [reply src/Organization/Behat/OrganizationApplicationContext.php]: Customized exceptins are good idea! Tnx!

Comment (msvrtan) [reply src/Organization/Behat/OrganizationApplicationContext.php]: Fixed!

**Comment by HcsOmot** [comment]
Great, now we just need to find those stealthy ninjas, hiding in the plain site... 🤔😀

**Comment by HcsOmot** [comment]
Hint - they're hiding in the feature file 🕵️‍♂️🤺👀

File: features/managing-organization-organizers.feature (hunk 2 above)
Comment (msvrtan) [reply features/managing-organization-organizers.feature]: Fixed

**Pull request merged**